mkdocs.yml

```diff
@@ -1,3 +1,19 @@
 site_name: DS Search Docs
 repo_url: https://github.com/nationalarchives/ds-search
 dev_addr: 0.0.0.0:65532
+theme:
+  name: material
+  font: false
+markdown_extensions:
+  - sane_lists
+  - toc:
+      permalink: true
+nav:
+  - "Home": "index.md"
+  - "Developer guide":
```

Comment:
Isn't the whole docs a "developer guide"? Would it make more sense to move these to the top level of the docs?

Reply:
Its a transfer from `ds-wagtail` of some bits as is or with some modifications.

In ds-wagtail [docs ](https://nationalarchives.github.io/ds-wagtail/) structure, the Home page called the overall documentation as `Technical Documentation`. They _separate_ Developer guide _contents_ from those in `Infrastructure and environments` and `Unique features` contents.

I don't know what you mean by move these to top level. The topmost level is `Home` here.

Reply:
What I mean was moving files like `docs/developer-guide/backend.md` to just `docs/backend.md` and updating this `mkdocs.yml` to something simpler like:

```
nav:
  - "Home": "index.md"
  - "Project conventions": "project-conventions.md"
  - "Contributing code": "contributing.md"
  - "Dependency management": "dependency-management.md"
  - "Local development gotchas": "local-development-gotchas.md"
  - "Backend development": "backend.md"
  - "Frontend development": "frontend.md"
```

It was just an idea to try and simplify the docs structure.

Reply:
removed `developer-guide` structure.